cypress/support/config/settings.js

```diff
@@ -3699,7 +3699,8 @@ module.exports = () => ({
           },
           test: {
             paths: ['/kyrgyz/topics/cvpv9djp9qqt?renderer_env=test'],
-            enabled: false,
+            enabled: false, // cannot use the above path as it fails on test that check the number of items on topic-promos. This is because
```

Comment:
Do we think there's a way in which we could get this working?

Reply:
Update: we need to leave this commented out for now, because we need the Hierarchical Grid fix on test. Once on test, we will enable this to see if the test page is working.